src/go/rpk/pkg/cli/cmd/cluster/config/set.go

```diff
@@ -84,7 +84,9 @@ If an empty string is given as the value, the property is reset to its default.`
 				// Special case 400 (validation) errors with friendly output
 				// about which configuration properties were invalid.
 				if he.Response.StatusCode == 400 {
-					fmt.Fprint(os.Stderr, formatValidationError(err, he))
+					ve, err := formatValidationError(err, he)
+					out.MaybeDie(err, "error setting config: %v", err)
+					fmt.Fprint(os.Stderr, ve)
 					out.Die("No changes were made.")
```

Comment:
How about merging the last line and this one into `No changes were made: %v.`?

Reply:
Same as the import one, only that this one is in the Run command so it's fine to leave it "Dying" here.

Reply:
Changed.